packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -131,11 +135,111 @@ export class TriggerManager extends Component {
       this.handleError(error)
     }
   }
+
+  getSelectedCipherId() {
+    const { selectedCipher } = this.state
+    return selectedCipher && selectedCipher.id
+  }
+
+  async getNewCipherId(login, password) {
+    const { vaultClient, konnector } = this.props
+
+    const konnectorURI = get(konnector, 'vendor_link')
+    const konnectorName = get(konnector, 'name') || get(konnector, 'slug')
+
+    const cipherData = {
+      id: null,
+      type: 1,
+      name: konnectorName,
+      login: {
+        username: login,
+        password,
+        uris: konnectorURI ? [{ uri: konnectorURI, match: 0 }] : []
```

Comment:
what is match:0 ?

Reply:
Maybe @y-lohse or @edas have the answer? I just moved this part of the code in a refactor, and have no idea why we have this `match: 0` here actually.

Reply:
Right, I missed that. It's the type of `uri` that we are using — full URL match, regular expression, etc. It's an ENUM, `0` presumably means "Full URL match" (@edas can you confirm?).

I think we can replace this with a more meaningful constant.

Reply:
from the jslib/enums/uriMatchType.ts: 

```typescript
export enum UriMatchType {
    Domain = 0,
    Host = 1,
    StartsWith = 2,
    Exact = 3,
    RegularExpression = 4,
    Never = 5,
}
``` 

0 means "match only with the domain part", which is the default.

Agree that we should probably have a constant here. I can export the enum from cozy-keys-lib so you can reuse it. Would it make sense ?

Reply:
In https://github.com/cozy/cozy-keys-lib/pull/25